configuration/pih/scripts/global/zl.js

```diff
@@ -88,3 +88,62 @@ function setUpExpandableContacts(badPhoneNumberMsg) {
 
 }
 
+function datepickerStartAndEndDateValidation(badDateInTheFutureMsg,badStartDateGreaterThanEndDateMsg){
```

Comment:
As with the other suggestion, use verbs for function names. The convention for these functions we use in HTMLForms is `setUp...`.

```suggestion
function setUpDatepickerStartAndEndDateValidation(badDateInTheFutureMsg, badStartDateGreaterThanEndDateMsg){
```